src/data/const.js

```diff
@@ -0,0 +1,29 @@
+const constant = {
```

Comment:
you could either name it constantS and assign to it Object.freeze({WEATHER_API_KEY... })

...or just enumerate them all like :

export const WAETHER...
export const CITY
export const PLACES

Reply:
I will make an export of all constants separately